File: code-alongs/04-ruby/99. warmups/the_board.rb

def create_board (rows, columns)
    row_1 = "# " * columns + "\n"
    row_2 = " #" * columns + "\n"

    if (rows % 2 == 0) 
        board = (row_1 + row_2) * (rows/2) 
    else
        board = (row_1 + row_2) * ((rows-1)/2) + row_1 
    end
end

puts "5,5"
print create_board(5,5)
puts "4,4"
print create_board(4,4)

File: code-alongs/01-javascript/98 - warm ups/theBoard.js
function createBoard(rows, columns) {
    row1 = ("# ").repeat(columns) + "\n"
    row2 = (" #").repeat(columns) + "\n"
    board = (row1 + row2).repeat(Math.floor(rows / 2))
    return (rows % 2 === 0 ? board : board + row1)
}